R/bs-theme.R

```diff
@@ -352,15 +352,7 @@ bootstrap_bundle <- function(version) {
 
   sass_bundle(
     main_bundle,
-    # color-contrast() was introduced in Bootstrap 5.
-    # We include our own version for a few reasons:
-    # 1. Easily turn off warnings options(bslib.color_contrast_warnings=F)
-    # 2. Allow Bootstrap 3 & 4 to use color-contrast() in variable definitions
-    # 3. Allow Bootstrap 3 & 4 to use bs_get_contrast()
-    sass_layer(
-      functions = sass_file(path_inst("sass-utils/color-contrast.scss")),
-      rules = sass_file(path_inst("bslib-scss", "bslib.scss"))
-    )
+    bslib_bundle()
```

Comment:
This is nice, thanks for abstracting this out! I also like that you moved the comments next to the implementation, that's helpful.